charts/rancher-vsphere-cpi/values.yaml

```diff
@@ -1,3 +1,6 @@
+# configEnabled controls whether the vSphere cloud provider ConfigMap is created.
+configEnabled: true
```

Comment:
I don't think this is correct For the credentials secret we have:
```yaml
vCenter:
  credentialsSecret:
    name: "vsphere-cpi-creds"
    generate: true
```

We should use the same structure for the cloud-config configmap configuration, instead of adding a random bool key to the top-level values that doesn't match any of the existing conventions for allowing users to manage their own resources.